python/dgl/graphbolt/sampled_subgraph.py

```diff
@@ -119,27 +117,33 @@ def exclude_edges(
 
         Examples
         --------
-        >>> node_pairs = {"A:relation:B": (torch.tensor([0, 1, 2]),
-        ...     torch.tensor([0, 1, 2]))}
+        >>> import dgl.graphbolt as gb
+        >>> import torch
+        >>> node_pairs = {"A:relation:B": gb.CSCFormatBase(
```

Comment:
nit: single quote or double quote?

Reply:
Double quote. It should use double quote in code, while string printed will be single quote.